pyproject.toml

```diff
@@ -1,3 +1,62 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[tool.flit.module]
+name = "aiida_cp2k"
+
+[project]
+name = "aiida-cp2k"
+dynamic = ["version", "description"]
+readme = "README.md"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Framework :: AiiDA",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: MacOS :: MacOS X",
+    "Programming Language :: Python :: 3",
+]
+dependencies = [
+    "aiida-core>=1.1.0,<2.0.0",
+    "aiida-gaussian-datatypes",
+    "ase",
+    "ruamel.yaml>=0.16.5",
+    "cp2k-output-tools",
+]
+
+[[project.authors]]
+name = "The AiiDA team"
+
+[project.urls]
+Homepage = "https://github.com/aiidateam/aiida-cp2k"
+
+[project.optional-dependencies]
+dev = [
+    "pgtest~=1.3",
+    "pytest~=6.0",
+    "pytest-cov~=2.11.1",
+    "coverage",
+    "pre-commit~=2.19",
+]
+docs = [
+    "sphinx",
+    "sphinx-rtd-theme",
+    "sphinxcontrib-contentui",
+    "sphinxcontrib-details-directive",
+]
+
+[project.entry-points."aiida.calculations"]
+cp2k = "aiida_cp2k.calculations:Cp2kCalculation"
```

Comment:
Does the identifier need to be quoted? That's what I have in my pyproject.toml's but not sure if it is required

Reply:
I don't know 🤷